example/helloWorld/src/helloworld.cpp

```diff
@@ -0,0 +1,177 @@
+/**
+ * \file
+ * Copyright 2014-2015 Benjamin Worpitz
```

Comment:
there is also a \copyright in doxygen, if you feel fancy. but that's more for a per-project level